_sass/components/_accordions.scss

```diff
@@ -82,16 +82,13 @@
         display: inline-block;
       }
 
-      display: none; 
+      display: none;
```

Comment:
Expected item on line 85 to appear before line 81. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets